### DISCUSSION TITLE
Only one of the classes is being applied

### DISCUSSION THREAD

**Discussion by s0h311** [tailwindlabs/tailwindcss#12151] [Help] [answered]
Hello everybody,

Somehow is the `h-60` class not being applied, where as the `relative` class in the very same `div` is.
I've already tried removing `node_modules` and running `yarn install`.

```JSX
{imageData?.src && (
  <div className='relative h-60'>
    <Image
      className={`rounded-t-2xl mb-2 ${imageData.onClick ? 'cursor-pointer' : ''}`}
      style={{ objectFit: 'cover' }}
      src={imageData.src}
      fill
      sizes='(max-width: 414px) 80vw, (max-width: 768px) 60vw, 20vw'
      quality={70}
      alt={imageData.alt}
      onClick={imageData.onClick}
    />
  </div>
)}
```

Here are screenshots of firefox inspector

![grafik](https://github.com/tailwindlabs/tailwindcss/assets/113988347/58ac06c9-fd5d-46c8-86cf-87c4148c4d2b)
![grafik](https://github.com/tailwindlabs/tailwindcss/assets/113988347/c1f60c79-baa0-4879-b7a2-f1eeaec4661d)

```
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 22.6.0: Fri Sep 15 13:41:28 PDT 2023; root:xnu-8796.141.3.700.8~1/RELEASE_ARM64_T6000
Binaries:
  Node: 18.16.1
  npm: 9.7.2
  Yarn: 3.6.1
  pnpm: N/A
Relevant Packages:
  next: 13.5.3
  eslint-config-next: 13.4.19
  react: 0.0.0-experimental-dddfe6882-20231005
  react-dom: 0.0.0-experimental-dddfe6882-20231005
  typescript: 5.2.2
Next.js Config:
  output: N/A
```

**Comment by alex-krasikau** [comment]
The code looks valid, can you please send the Tailwind config? Make sure that you don't overwrite the spacing configuration.

Here's the working example on the default configuration - https://tailwindcss.com/docs/customizing-spacing.

**Reply by s0h311** [reply]
Hi @alex-krasikau, here is my config

```TypeScript
/** @type {import('tailwindcss').Config} */

const tailwindConfigs = {
  content: [
    './pages/**/*.{js,ts,jsx,tsx,mdx}',
    './components/**/*.{js,ts,jsx,tsx,mdx}',
    './app/**/*.{js,ts,jsx,tsx,mdx}',
  ],
  theme: {
    extend: {
      colors: {
        primary: 'var(--primary)',
        secondary: 'var(--secondary)',
        accent: 'var(--accent)',
        bgColor: 'var(--bg-color)',
        textColor: 'var(--text-color)',
      },
      width: {
        screen: '100dvw',
      },
      height: {
        screen: '100dvh',
      },
    },
  },
  plugins: [],
}

export default tailwindConfigs

```

**Reply by s0h311** [reply, marked as answer]
@alex-krasikau I replaced `'./pages/**/*.{js,ts,jsx,tsx,mdx}',`  with `'./ui/**/*.{js,ts,jsx,tsx,mdx}',`. Since that's the directory where my component is and that seemed to solve it. I didn't use the Next.js Pages Router anyway.
